topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/workflows/nanopore_preprocessing.ga

```diff
@@ -50,8 +50,8 @@
             "name": "Input dataset",
             "outputs": [],
             "position": {
-                "left": 0,
-                "top": 600.9666748046875
+                "left": 0.0,
```

Comment:
You can restore this whole file. It only contains non-significant changes now.

Reply:
I will re download the workflow and put it

Reply:
You just need to git restore the file :-)